andino_gz/worlds/office.sdf

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='ASCII'?>
 <sdf version='1.7'>
-  <world name='gazebo_world'>
```

Comment:
actually remove this change. We need it to be named gazebo_world due to some logic later on when having multiple robots

Reply:
done